web/src/main/ui/app/components/flows-new/services/manage-flows.service.ts

```diff
@@ -100,4 +100,10 @@ ManageFlowsService {
     console.log('GET api/current-project/mappings/functions');
     return this.http.get('api/current-project/mappings/functions');
   }
+
+  getDefaultCollections(entityName) {
```

Comment:
Is manage-flows.service kind of a broad facade across DHF? Just curious, as I don't think of "default merge collections" as belonging to a "manage flows" service, but if it's intended to be a pretty broad facade, that's fine.

Reply:
Its surface area is broad right now. That's where all the services supporting step-based flows live. I agree that it should subdivided at some point.